### PR TITLE
Minor clean up based on Scrutinizers results

### PR DESCRIPTION
**Pull request by ameech**

### CODE REVIEW

**Comment by elazar** [comment]
:+1:

File: src/Router.php [hunk 1]
```diff
@@ -162,13 +162,11 @@ public function dispatch($method, $path)
...
             case Dispatcher::NOT_FOUND:
                 throw new HttpNotFound;
-                break;
             case Dispatcher::METHOD_NOT_ALLOWED:
                 throw (new HttpMethodNotAllowed)
                     ->setAllowedMethods($routeInfo[1]);
-                break;
             case Dispatcher::FOUND:
-                list($_, $route, $arguments) = $routeInfo;
+                list(, $route, $arguments) = $routeInfo;
```
Comment (shadowhand) [inline src/Router.php]: :whale: TIL

**Comment by shadowhand** [comment]
Let's rebase this and get it merged. :)

**Comment by dolfelt** [comment]
@ameech Can you get this merged in?

**Comment by shadowhand** [comment]
@ameech I'll take it.

**Pull request merged**